ts/ts-bas/src/main/java/se/inera/intyg/common/ts_bas/v7/model/converter/certificate/question/QuestionLakarintygAvOgonspecialistMessage.java

```diff
@@ -68,4 +70,13 @@ public static CertificateDataElement toCertificate(int index, CertificateTextPro
             )
             .build();
     }
+
+    private static Boolean setVisibility(Syn syn) {
```

Comment:
Consider naming it getVisibility(..) instead of set, as the code doesn't set anything.